hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesCapacitySched.java

```diff
@@ -135,7 +137,7 @@ private static void setupQueueConfiguration(
 
     // Define top-level queues
     config.setQueues(CapacitySchedulerConfiguration.ROOT,
-        new String[] {"a", "b"});
+        new String[] {"a", "b", "d"});
```

Comment:
This test is already convoluted as it is. If it is possible, setting max-parallel-apps on an existing queue would be better.

Reply:
Done.